### PR TITLE
Use the latest tools common version

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
   "devDependencies": {
     "gulp": "^3.8.10",
-    "tools-common": "github:rictic/tools-common#0de693a",
+    "tools-common": "github:Polymer/tools-common#ed1dd8f",
```
Comment (justinfagnani) [inline package.json]: is this just until you publish a new version to npm?

**Comment by justinfagnani** [comment]
Oh, maybe this is what we need to get it building via `gulp build`...

**Comment by stramel** [comment]
I believe this has already been merged to master at some point.

**Comment by justinfagnani** [comment]
Looks like it.

**Pull request closed without merging**